core/services/ardupilot_manager/mavlink_proxy/MAVLinkRouter.py

```diff
@@ -29,11 +29,23 @@ def convert_endpoint(endpoint: Endpoint) -> str:
                 return f"--tcp-port {endpoint.argument}"
             if endpoint.connection_type == EndpointType.TCPClient:
                 return f"--tcp-endpoint {endpoint.place}:{endpoint.argument}"
+            if endpoint.connection_type == EndpointType.UDPServer:
+                return f"{endpoint.place}:{endpoint.argument}"
             if endpoint.connection_type == EndpointType.UDPClient:
                 return f"--endpoint {endpoint.place}:{endpoint.argument}"
             raise ValueError(f"Endpoint of type {endpoint.connection_type} not supported on MavlinkRouter.")
 
-        endpoints = " ".join([convert_endpoint(endpoint) for endpoint in self.endpoints()])
+        types_order = {
```

Comment:
A comment about why the order is important shoud be added here since this is not a clear thing.